### ISSUE TITLE
Using a / in a Goal name

### ISSUE THREAD

**Issue by D-Dev** [failys/cairis#240]
### Expected ? Actualbehaviour
Having used a / in a Goal name, when I go to the list of goals and select this example to edit it, the next information screen does not generate/open as expected. Selecting any other Goal continues to work as expected. No error appears in the terminal though.

### Steps to reproduce the behaviour
<goal name="(C-F2) Allow for safe Air MEDEVAC landing at FOB/FST" originator="D">

### Version of CAIRIS
Current

### Operating Systems / Hardware 
Win7 - VMw - Linux Ub17 - Chrome

**Comment by failys** [comment]
The problem is that, when retrieving stored goals (and any other object), the object name is included in the API call, i.e. /api/goals/name/goalname.  Consequently, adding the / (even when encoded) leads to a different API path.

I've updated the UI to forbid the addition of 'reserved characters' to object names.  I've reflected this in the documentation: https://cairis.readthedocs.io/en/latest/reserved.html
